client/my-sites/plans/current-plan/current-plan-thank-you-card/index.js

```diff
@@ -1,91 +1,129 @@
 			: null;
+
 		return (
-			<Card className="current-plan-thank-you-card">
-				<QuerySitePurchases siteId={ siteId } />
-				<div className="current-plan-thank-you-card__content">
-					<img
-						className="current-plan-thank-you-card__illustration"
-						alt=""
-						aria-hidden="true"
-						src="/calypso/images/illustrations/fireworks.svg"
-					/>
-					<h1 className="current-plan-thank-you-card__title">
-						{ translate( 'Thank you for your purchase!' ) }
-					</h1>
-					<p>
-						{ duration && planName
-							? translate(
-									'Your website is on a %(planName)s plan for %(duration)s. That means it has lots of useful security tools — let’s walk through a short checklist of the essentials so Jetpack can start monitoring things for you.',
-									{
-										args: { duration, planName },
-									}
-							  )
-							: ' ' /* &nbsp; maintain some space */ }
-					</p>
-					<p>
-						{ translate(
-							'We’ve taken the liberty of starting the first two items, since they’re key to your site’s safety: we’re configuring spam filtering and backups for you now. Once that’s done, we can work through the rest of the checklist.'
+			<Fragment>
+				<img
+					className="current-plan-thank-you-card__illustration"
+					alt=""
+					aria-hidden="true"
+					src="/calypso/images/illustrations/fireworks.svg"
+				/>
+				<h1 className="current-plan-thank-you-card__title">
+					{ translate( 'Thank you for your purchase!' ) }
+				</h1>
+				<p>
+					{ duration &&
+						planName &&
+						translate(
+							'Your website is on a %(planName)s plan for %(duration)s. That means it has lots of useful security tools — let’s walk through a short checklist of the essentials so Jetpack can start monitoring things for you.',
+							{
+								args: { duration, planName },
+							}
 						) }
-					</p>
-					{ /* can be 0 */ isFinite( this.props.progressComplete ) &&
-						/* shouldn't be 0 */ this.props.progressTotal && (
-							<ProgressBar
-								isPulsing
-								total={ this.props.progressTotal }
-								value={ this.props.progressComplete }
-							/>
-						) }
-					<p>
-						<a
-							className="current-plan-thank-you-card__skip-setup"
-							href={ /* @TODO (sirreal) fix this */ document.location.pathname }
-						>
-							{ translate( 'Skip setup. I’ll do this later.' ) }
-						</a>
-					</p>
-				</div>
-			</Card>
+				</p>
+				<p>
+					{ translate(
+						'We’ve taken the liberty of starting the first two items, since they’re key to your site’s safety: we’re configuring spam filtering and backups for you now. Once that’s done, we can work through the rest of the checklist.'
+					) }
+				</p>
+
+				<ProgressBar isPulsing total={ 100 } value={ progressComplete || 0 } />
+
+				<p>
+					<a href={ this.getMyPlanRoute() }>{ translate( 'Skip setup. I’ll do this later.' ) }</a>
+				</p>
+			</Fragment>
+		);
+	}
+
+	renderSuccess() {
+		const { translate } = this.props;
+
+		return (
+			<Fragment>
```

Comment:
Yeah, exactly because of `renderSetup()` I chose to go that way. We could add `renderSuccess()` to `render()` but since `renderSetup()` would still be its own method, it wouldn't look much better IMHO.

Anyway, after we shortened the copy in 7bea25ce8afbc6c7e3667e5fcc9d2a203387ae5e and got rid of a bunch of things, the JSX ended up being pretty simple in both methods. So in f909774cb17782df04c7cd9e60c3e26c644cc606 I moved it all to `render()`.